### ISSUE TITLE
module_user_guide.ipynb: NegativeBinomial function call raises "unexpected keyword argument 'total_counts'" error

### ISSUE THREAD

**Issue by rangan2510** [YosefLab/scvi-tutorials#39]
I am trying to create my own PyTorch module by looking at the documentation as in [Crafting the Module in vanilla PyTorch](https://docs.scvi-tools.org/en/stable/tutorials/notebooks/module_user_guide.html#Crafting-the-Module-in-vanilla-PyTorch). In the next step, I want to implement the model as in [Constructing a high-level model](https://docs.scvi-tools.org/en/stable/tutorials/notebooks/model_user_guide.html). 

In the first linked page, the `loss` function has a line, in cell 6, 
`log_lik = NegativeBinomial(total_counts=theta, total=nb_logits).log_prob(x).sum(dim=-1)`

whereas the [PyTorch documentation](https://pytorch.org/docs/1.2.0/distributions.html#torch.distributions.negative_binomial.NegativeBinomial) for the function is given as
`torch.distributions.negative_binomial.NegativeBinomial(total_count, probs=None, logits=None, validate_args=None)` 

So, in the next stage, when I am replacing the built-in `VAE` model with `MyModule`, I am getting the `unexpected keyword argument 'total_counts'` error.

I am attaching the file for reproducing the issue. 

[scvi_tut.py.txt](https://github.com/YosefLab/scvi-tutorials/files/7789894/scvi_tut.py.txt)

I am using PyTorch 1.10.0+cpu.

**Comment by watiss** [comment]
Hmm it appears to be a typo? I put out a PR to fix. Thanks for reporting!

**Comment by rangan2510** [comment]
"total" is also an unexpected keyword here. Review the attached code file once where I was using a user module on a 10x genomics file.
